content/security/reporting.adoc

```diff
@@ -58,6 +58,24 @@ We do not consider the following issues to be vulnerabilities in Jenkins (core +
 * Jobs started by a specific user can run on agents where the user lacks Agent/Build permission and can themselves trigger builds of jobs where the user lacks Job/Build permission.
   link:/doc/book/security/build-authorization/[See the documentation on Access Control for Builds].
 
+
+=== CVEs in dependencies
+
+In the case of CVEs found in third party dependencies included in the Jenkins project, if the ticket does not include reproduction steps, a proof or at least a good argument, we are closing them.
+Those CVEs are internally analysed and most of the time the project is not impacted.
+
+When a CVE has an impact, we are including them in an advisory, like link:/security/advisory/2022-09-09/#SECURITY-2868[CVE-2022-2048 in Jetty] or link:/security/advisory/2022-02-09/#SECURITY-2602[CVE-2021-43859 in XStream].
```

Comment:
```suggestion
When a CVE has an impact to the security of Jenkins, we include it in an advisory, like link:/security/advisory/2022-09-09/#SECURITY-2868[CVE-2022-2048 in Jetty] or link:/security/advisory/2022-02-09/#SECURITY-2602[CVE-2021-43859 in XStream].
```

Some CVEs have an "impact", but if it's throwing a runtime exception, we don't really care.